scripts/modules/integrations/linear-integration-handler.js

```
@@ -281,76 +482,1850 @@ export class LinearIntegrationHandler extends BaseIntegrationHandler {
+			case 'error':
+				logLevel = 'error';
+				break;
+			case 'retry':
+				logLevel = 'warn';
+				break;
+			case 'progress':
+				logLevel = 'debug';
+				break;
+			case 'success':
+			default:
+				logLevel = 'info';
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)